fix/stuck-instruments.lua

```diff
@@ -0,0 +1,65 @@
+-- Fixes instruments that never got played during a performance
+
+local help = [====[
+
+fix/stuck-instruments
+=====================
+
+Fixes instruments that were picked up for a performance, but were instead
+simulated and are now stuck permanently in a job that no longer exists.
+
+This works around the issue encountered with :bug:`9485`, and should be run
+if you notice any instruments lying on the ground that seem to be stuck in a
+job.
+
+Run ``fix/stuck-instruments -n`` or ``fix/stuck-instruments --dry-run`` to
+list how many instruments would be fixed without performing the action.
+
+]====]
```

Comment:
help text should go in docs/fix/stuck-instruments.rst. Check out other files in that directory for the documentation structure. Also more info here: https://docs.dfhack.org/en/latest/docs/dev/Documentation.html